content/_help/get-started/authentication-options._fr.md

```diff
@@ -16,7 +16,7 @@ redirect_from:
   - /fr/help/authentication-methods/which-authentication-method-should-i-use/
   - /fr/help/authentication-methods/
 ---
-En plus de votre mot de passe, Login.gov exige que vous mettiez en place au moins une méthode d'authentification secondaire pour assurer la sécurité de votre compte. Il s'agit d'une authentification à deux facteurs (A2F). Nous utilisons l'A2F comme un niveau de protection supplémentaire pour sécuriser vos informations.
+En plus de votre mot de passe, Login.gov exige que vous mettiez en place au moins une méthode d'authentification pour assurer la sécurité de votre compte. Il s'agit d'une authentification multifactorielle (AMF). Nous utilisons l'AMF comme un niveau de protection supplémentaire pour sécuriser vos informations.
```

Comment:
The other languages shuffle around the "For or touch unlock" and "PIV or CAC for federal government employees and military" sections. Did we want to do that for the French content was well?

Reply:
> The other languages shuffle around the "For or touch unlock" and "PIV or CAC for federal government employees and military" sections. Did we want to do that for the French content was well?

Thanks for the catch. It's been fixed!

Reply:
I only see "Face or touch unlock" has moved, not PIV.

Reply:
> I only see "Face or touch unlock" has moved, not PIV.

Argh! Thanks for letting me know. ... it should all be fixed now.